src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/DebugFuncEval.cs

```diff
@@ -0,0 +1,143 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Reflection.Runtime.General;
+
+using Internal.TypeSystem;
+using Internal.Runtime.Augments;
+using Internal.Runtime.CallInterceptor;
+using Internal.TypeSystem.NativeFormat;
+using Internal.NativeFormat;
+
+namespace System.Runtime.InteropServices
```

Comment:
Is there anything special about these that requires them to be in a public namespace?